library_schemas/score/prerelease/HED_score_1.0.0.mediawiki

```diff
@@ -871,18 +871,18 @@ This project was supported by the by the National Institute of Mental Health of
                     ***** Semiology-motor-eye-blinking
                     ***** Semiology-motor-other-elementary-motor
                         ****** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Semiology-motor-automatisms
```

Comment:
'Automatisms' is used in plural throughout the paper and software in accordance with the ILAE Glossary of Descriptive Terminology for Ictal Semiology (Blume et al., 2001).

Reply:
I'll change this back in the PR.